### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/shared-ui/locales/en-US.json [hunk 1]
```diff
@@ -401,7 +401,7 @@
     "plugin_red_packet_expired": "Expired",
     "plugin_red_packet_indivisible": "The minimum amount for each share is {{amount}} {{symbol}}",
     "plugin_nft_red_packet_data_broken": "The Lucky Drop can’t be sent due to data damage.",
-    "plugin_gitcoin_readme": "By using this service, you will also be contributing 5% of your contribution to the <fund>Gitcoin grants development fund</fund>.",
+    "plugin_gitcoin_readme": "By using this service, you will also be contributing 5% of your contributions to the <fund>Gitcoin grants development fund</fund>.",
```
Comment (guanbinrui) [inline packages/mask/shared-ui/locales/en-US.json]: Revert it.

**Comment by wenluomask** [comment]
Checked.

**Comment by guanbinrui** [comment]
> Checked.

No. He didn't do anything related to the issue.

**Comment by wenluomask** [comment]
> > Checked.
> 
> No. He didn't do anything related to the issue.

I thought what you mean in Jira is Gitcoin plugin. For the Donation tab, we still need to wait for confirmation.

**Comment by wenluomask** [comment]
Checked.

**Review by Jack-Works — DISMISSED** [review]

**Review by guanbinrui — DISMISSED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**